partner_location_auto_create/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# © 2015 Savoir-faire Linux
+# © 2018 brain-tec AG (Kumar Aberer <kumar.aberer@braintec-group.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Partner Location Auto Create',
+    'version': '11.0.1.0.0',
+    'author': 'brain-tec AG, Savoir-faire Linux,Odoo Community Association ('
+              'OCA)',
+    'category': 'Warehouse',
+    'license': 'AGPL-3',
+    'complexity': 'normal',
+    'images': [],
+    'website': 'http://www.savoirfairelinux.com',
```

Comment:
Website should be https://github.com/OCA/stock-logistics-warehouse

Reply:
:+1: